interfaces/handlerInterface.go

```diff
@@ -0,0 +1,23 @@
+package interfaces
+
+// These are the messages read off and written into the websocket. Since this
+// struct serves as both read and write, we include the "Id" field which is
+// required only for writing.
+type Message struct {
+	ID      uint64 `json:"id"`
+	Type    string `json:"type"`
+	Channel string `json:"channel"`
+	Text    string `json:"text"`
+}
+
+// This can post a message to slack.
```

Comment:
comment on exported type Postable should be of the form "Postable ..." (with optional leading article)